src/tools/compiletest/src/lib.rs

```diff
@@ -477,6 +491,14 @@ pub fn parse_config(args: Vec<String>) -> Config {
     }
 }
 
+fn parse_bool_option(value: &str) -> Option<bool> {
+    match value {
+        "off" | "no" | "n" | "false" => Some(false),
+        "on" | "yes" | "y" | "true" => Some(true),
+        _ => None,
+    }
+}
```

Comment:
Nit: can we _just_ support either {on,off} or {true,false} and not all of them

<img width="80" height="60" alt="ferrisClueless-80x60" src="https://github.com/user-attachments/assets/0f7582df-1269-4cb1-b553-be1697482314" />

Reply:
That's fair.